### PR TITLE
Assorted OpenDistro and OpenSearch fixes

### PR DESCRIPTION
**Pull request by tobybellwood**
This PR does the following:

* adds `rejectUnauthorized: false` to the got commands to interact with Kibana and OpenDistro/Opensearch security - Whilst the elasticsearch plugin can handle the varied authentication, got can't - so we need this. Given this comminication is within the cluster, and to endpoints defined within the API service, the risk of setting this is minimal
* adds the `'osd-xsrf': 'true'` header to the kibanaClient to handle OpenSearch Dashboards, as well as Kibana
* adds `indices:monitor/settings/get` to intial index permissions for groups to enable reporting

### CODE REVIEW

File: local-dev/lagoon-logging.values.yaml
Comment (smlx) [inline local-dev/lagoon-logging.values.yaml]: I guess this should be removed before merge?

File: services/api/src/resources/group/opendistroSecurity.ts
Comment (smlx) [inline services/api/src/resources/group/opendistroSecurity.ts:48]: What does this change allow?

**Review by smlx — APPROVED** [review]
I have no way to test this, but LGTM otherwise. Also goes some way towards fixing https://github.com/uselagoon/lagoon/issues/2844

File: services/api/src/resources/group/opendistroSecurity.ts
Comment (tobybellwood) [reply services/api/src/resources/group/opendistroSecurity.ts:48]: This is one of the core permissions required to add reporting access to user accounts in ODFE. Without this permissions, it appears that users are unable to select the data needed to create reports.

File: local-dev/lagoon-logging.values.yaml
Comment (tobybellwood) [reply local-dev/lagoon-logging.values.yaml]: ah yes - that's the merged PR for fluent plugin updates #3009

**Pull request merged**
